src/main/java/dk/sdu/se_f22/sharedlibrary/db/database.sql

```diff
@@ -0,0 +1,40 @@
+
+/*
+ Drop alle tabellerne i starten af dokumentet.
+    -   Drop tabeller som refererer til andre tabeller
+    -   Drop resten af tabellerne
+ */
+ -- Drop dependent tables
+DROP TABLE IF EXISTS BrandProductTypeJunction;
+
+-- Drop all other tables
+DROP TABLE IF EXISTS Brand;
+DROP TABLE IF EXISTS ProductType;
+DROP TABLE IF EXISTS config;
+
+/*
+ Her oprettes tabellerne, der skal ikke INSERT INTO tabellerne endnu, da vi vil lave en .java fil som seeder hele databasen på én gang,
+ og kalder hver gruppes seedDatabase()-metode
+ */
+CREATE TABLE Brand(
+    id           serial PRIMARY KEY,
+    name         VARCHAR(255) UNIQUE NOT NULL,
+    description  TEXT,
+    founded      VARCHAR(255),
+    headquarters VARCHAR(255)
+);
+
+CREATE TABLE ProductType(
+    id   serial PRIMARY KEY,
+    type VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE BrandProductTypeJunction(
+    brandId   INTEGER REFERENCES Brand (id),
+    productId INTEGER REFERENCES ProductType (id)
+);
+
+CREATE TABLE Config(
+    brandIndexInterval INTEGER NOT NULL
```

Comment:
For this @v-nemeth should comment, he made those canges.

Reply:
Sorry for the late reply

The thought was to have one single document with all the SQL required to make the database-tables, which every group can access. The thought was that everyone should be able to run the database.sql and be up-to-date with what was on GitHub. Changes made to the database should therefore be added to this document, and made so that the database is first deleted, and then created again.

After the database is created again, you should run a seedDatabase()-method containing all the different-group-methods for inserting data into the database.
This has been discussed with our guidance counselor as well as group 2.1.

Reply:
We need to discuss how we want to do that and how we're going to structure it